tidb-cloud/terraform-use-import-resource.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Use Import Resource
+summary: Learn how to manage the import task using the import resource.
+---
+
+# Use Import Resource
+
+Import resource supports:
```

Comment:
```suggestion
You can learn how to import data to a TiDB Cloud cluster with the `tidbcloud_import` resource in this document.

The features of the `tidbcloud_import` resource include the following:
```

Reply:
change reason: This is to use a similar introductory sentence as other Terraform integration docs.